hello.java

```diff
@@ -0,0 +1,15 @@
+import java.util.*;
+
+public class Hello
+{
+  public static int main(String[] args)
```

Comment:
Is `String[] args` necessary in this case? The program takes no arguments.

Reply:
Yes, `String[] args` is necessary for the main  function, kinda part of the syntax as mentioned [here](https://docs.oracle.com/javase/specs/jls/se8/html/jls-12.html#jls-12.1.4:~:text=Finally%2C%20after%20completion,declarations%20is%20acceptable%3A)

Reply:
Good ol' Java

Reply:
my bad though, `main` should have been `void`, instead of `int`.

Reply:
No need to worry about anything.